packages/jest-reporters/src/__tests__/GithubActionsLogsReporter.test.js

```diff
@@ -0,0 +1,416 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// copied from https://github.com/MatteoH2O1999/github-actions-jest-reporter/blob/master/tests/gha.reporter.test.js
+
+import util from 'util';
```

Comment:
I think this test file should be written in TypeScript and it would be the best to move the tests into `GitHubActionsReporter.test.ts`. For instance, `utils` are imported here, but not used. Also line `const testContext = {};` is repeating few times in this file, but `testContext` variable is unused. TS helps to clean this up.

Reply:
Merged